config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -0,0 +1,123 @@
+#
+#   Copyright (c) 2022 Project CHIP Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+# The purpose of this file is to define Kconfig options enabling specific features.
+# Feature options will define defaults for settings that are related to the specific feature.
+
+if CHIP
+
+config CHIP_QSPI_NOR
+	bool "Enable QSPI NOR feature set"
+	help
+		Enables QSPI NOR with the set of options
+		configuring pages and buffer sizes.
+
+if CHIP_QSPI_NOR
+
+config NORDIC_QSPI_NOR
+	bool
+	default y
+
+config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
```

Comment:
I think it's set by default based on DTS, so can potentially be removed now.

Reply:
@kkasperczyk-no ?

Reply:
Yes, that's right, fixed.